src/plugin.js

```diff
@@ -37,8 +37,16 @@ export default {
               }),
               {}
```

Comment:
No need to make a variable and spread after, you could add `{ ready: true}` here.

(also i prefer `ready` over `isReady`)

Reply:
oh yeah! forgot about the reducer